docs/mui-vale/styles/MUI/MuiBrandName.yml

```diff
@@ -3,7 +3,7 @@
 # For example, Apple does this meticulously with their brand name: https://www.apple.com/macbook-air/.
 # Also read https://www.chrisdpeters.com/blog/using-non-breaking-spaces-to-help-with-branding/ for why.
 extends: substitution
-message: Use a non-breaking space (option + space, instead of space) for brand name ('%s' instead of '%s')
+message: Use a non-breaking space (option+space on Mac or ctrl+shift+space on Windows, instead of space) for brand name ('%s' instead of '%s')
```

Comment:
I don't think it's a correct shortcut on Windows - it doesn't work for me. AFAIK you have to use `Alt + 0160`.

Reply:
Ctrl+Shift+space seems to be the key as shown [here](https://support.microsoft.com/en-us/office/keep-text-together-af94e5b8-3a5a-4cb0-9c53-dea56b43d96d#:~:text=the%20keyboard%20shortcut.-,Click%20where%20you%20want%20to%20insert%20the%20nonbreaking%20space,press%20Ctrl%2BShift%2BSpacebar.) in this [image](https://support.content.office.net/en-us/media/f6e9d321-a8fb-4746-acd1-0b3e97df18fe.png).

Reply:
True, it's very stupid especially if you don't have numeric keyboard, you have to use the On-Screen keyboard 🤦

Reply:
```suggestion
message: Use a non-breaking space (option+space on Mac or Alt+0160 on Windows, instead of space) for brand name ('%s' instead of '%s')
```